docsbox/config/config.yml

```diff
@@ -201,9 +201,9 @@ COMMON: &common
       system: 'Document Conversion Service'
       environment: 'DEV'
 
-  GHOSTSCRIPT_EXEC: ['gs', '-dPDFA', '-dBATCH', '-dNOPAUSE', '-sProcessColorModel=DeviceRGB', '-sDEVICE=pdfwrite', '-sPDFACompatibilityPolicy=1', '-dPDFSETTINGS=/printer']
+  GHOSTSCRIPT_EXEC: ['gs', '-dPDFA', '-dBATCH', '-dNOPAUSE', '-sProcessColorModel=DeviceRGB', '-sDEVICE=pdfwrite', '-dPDFACompatibilityPolicy=1', '-dPDFSETTINGS=/printer', '-dNumRenderingThreads=2', '-dBufferSpace=2000000000']
```

Comment:
We are already scaling the workers per cpu in live, not sure how well it would behave if we increase the threads for rendering for each worker. Might throttle it up